FinanceManager.Core/Entities/Accounts/BankAccount.cs

```diff
@@ -33,6 +17,39 @@ public BankAccount(int id, string name, AccountType accountType) : base(id, name
             AccountType = accountType;
             Entries = new List<BankAccountEntry>();
         }
+
+        public void Add(AddBankEntryDto entry)
+        {
+            Entries ??= new List<BankAccountEntry>();
+            var alredyExistingEntry = Entries.FirstOrDefault(x => x.PostingDate == entry.PostingDate && x.ValueChange == entry.ValueChange);
+            if (alredyExistingEntry is not null)
+            {
+                throw new Exception($"WARNING - Entry already exist, can not be added: Id:{alredyExistingEntry.Id}, Posting date{alredyExistingEntry.PostingDate}, " +
```

Comment:
Nie rzucaj domyślnej klasy excpetion, jeżeli możesz zrób swoja klasę wyjątku i użyj jej, poprawi to nawigacje w kodzie i od razu zakomunikuje co sie stalo nie tak